---
layout: archive
title: "Publications"
permalink: /publications/
author_profile: true
---

Pre-prints of note 
====

  * (O'Connor T, Grant CE, Boden M and Bailey TL. T-Gene: Improved target gene prediction.)[http://biorxiv.org/cgi/content/short/803221v1]
  * (Shim WJ, Sinniah E et al. Comparative analysis of diverse cell states establishes an epigenetic basis for inferring regulatory genes governing cell identity.) [https://www.biorxiv.org/content/10.1101/635516v1]
  * (Lai JS, Rost B, Kobe B and Bodén M. Evolutionary model of protein secondary structure capable of revealing new biological relationships.) [https://www.biorxiv.org/content/10.1101/563452v1]

Journal papers (since 2000) 
====

  * Horsefield S, et al. (2019) NAD+ cleavage activity by animal and plant TIR domains in cell death pathways. //Science// 365(6455):793-799. DOI [https://doi.org/10.1126/science.aax1911] [https://science.sciencemag.org/content/365/6455/793]
  * Foley G, Sützl L, D'Cunha SA, Gillam EMJ and Bodén M (2019). SeqScrub: A web tool for automatic cleaning and annotation of FASTA file headers for bioinformatic applications. //Biotechniques//. DOI [[https://doi.org/10.2144/btn-2018-0188|10.2144/btn-2018-0188]]
  * Littmann M, Goldberg T, Seitz S, Bodén M and Rost B (2019). Detailed prediction of protein sub-nuclear localization. //BMC Bioinformatics//. 20(1):205. DOI [[https://doi.org/10.1186/s12859-019-2790-9|10.1186/s12859-019-2790-9]]
  * Sützl L, Foley G, Gillam EM, Bodén M and Haltrich D (2019). The GMC family of oxidoreductases revisited: analysis and evolution of fungal GMC oxidoreductases. //Biotechnology for Biofuels//. 12:118. DOI [[https://doi.org/10.1186/s13068-019-1457-0|10.1186/s13068-019-1457-0]]
  * Fraser J, Essebier A, Brown A, Ayala Davila R, Sengar A, Tu A, Ensbey K, Day B, Scott M, Gronostajski R, Wainwright B, Bodén M,  Harvey T and Piper M. (2019) Granule neuron precursor cell proliferation is regulated by NFIX and intersectin 1 during postnatal cerebellar development. //Brain Structure and Function//. 224:2, pp 811–827. DOI [[https://doi.org/10.1007/s00429-018-1801-3|10.1007/s00429-018-1801-3]]
  * Gumulya Y, et al. (2019) Engineering thermostable CYP2D enzymes for biocatalysis using combinatorial libraries of ancestors for directed evolution (CLADE). //ChemCatChem//. 11:2, pp 841-850. DOI [[https://doi.org/10.1002/cctc.201801644|10.1002/cctc.201801644]]
  * Gumulya Y, et al. (2018) Engineering highly functional thermostable proteins using ancestral sequence reconstruction. //Nature Catalysis//. 1, pp 878–888. DOI [[https://doi.org/10.1038/s41929-018-0159-5|10.1038/s41929-018-0159-5]] [[https://rdcu.be/9Pia|Free access]]
  * Fletcher SJ, Bodén M, Mitter N, Carroll BJ (2018) SCRAM: a pipeline for fast index-free small RNA read alignment and visualization. //Bioinformatics//. 34:15, pp 2670–2672. DOI [[http://dx.doi.org/10.1093/bioinformatics/bty161|10.1093/bioinformatics/bty161]][[https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/bty161/4938488?guestAccessKey=2d418e5c-62e3-4608-af5e-b39b40c26ac3|Free access]]
  * Zaugg J, Gumulya Y, Bodén M, Mark AE, Malde AK (2018) Effect of Binding on Enantioselectivity of Epoxide Hydrolase. //Chem Inf Model//. 58 (3): 630–640. DOI [[https://doi.org/10.1021/acs.jcim.7b00353|10.1021/acs.jcim.7b00353]].
  * Zaugg J, Gumulya Y, Malde AK and Bodén M (2017) Learning Epistatic Interactions from Sequence-Activity Data to Predict Enantioselectivity. //J Comput Aided Mol Des//. 31(12):1085–1096. DOI [[https://doi.org/10.1007/s10822-017-0090-x|10.1007/s10822-017-0090-x]] [[http://rdcu.be/BdRu|Free access]]
  * Essebier A, Lamprecht M, Piper M and Bodén M (2017) Bioinformatics approaches to predict target genes from transcription factor binding data. //Methods//. Volume 131, Pages 111-119. DOI [[http://dx.doi.org/10.1016/j.ymeth.2017.09.001|10.1016/j.ymeth.2017.09.001]].
  * Bernhofer M, Goldberg T, Wolf S, Ahmed M, Zaugg J, Bodén M and Rost B (2017) NLSdb - major update for database of nuclear localization signals and nuclear export signals, //Nucleic Acids Res//. 46:D1, pp D503–D508. DOI [[http://dx.doi.org/10.1093/nar/gkx1021|10.1093/nar/gkx1021]]
  * Patrick R, Kobe B, Le Cao KA and Bodén M (2017) PhosphoPICK-SNP: Quantifying the effect of amino acid variants on protein phosphorylation, //Bioinformatics//. 33 (12): 1773-1781. DOI [[http://dx.doi.org/10.1093/bioinformatics/btx072|10.1093/bioinformatics/btx072]]. [[https://academic.oup.com/bioinformatics/article/33/12/1773/2982049/PhosphoPICKSNP-quantifying-the-effect-of-amino?guestAccessKey=eec63034-0aec-4fa6-b13a-7cbbb167c98d|Free access]]
  * O'Connor TR, Bodén M and Bailey, TL (2017) CISMAPPER: predicting regulatory interactions from transcription factor ChIP-seq data. //Nucleic Acids Res// 45 (4): e19. DOI: [[http://dx.doi.org/10.1093/nar/gkw956|10.1093/nar/gkw956]].
  * Williams SJ, Yin L, Foley G, Casey LW, Outram MA, Ericsson DJ, Lu J, Bodén M, Dry I, Kobe B (2016) Structure and function of the TIR domain from the grape NLR protein RPV1. //Frontiers in Plant Science//. DOI: [[http://dx.doi.org/10.3389/fpls.2016.01850|10.3389/fpls.2016.01850]]
  * Fraser J, Essebier A, Gronostajski RM, Bodén M, Wainwright BJ, Harvey TJ, Piper M (2016) Cell type-specific expression of NFIX in the developing and adult cerebellum. //Brain Structure and Function//. 222:5, pp 2251–2270. DOI: [[http://dx.doi.org/10.1007/s00429-016-1340-8|10.1007/s00429-016-1340-8]]
  * Harris L, Zalucki O, Gobius I, Osinski J, Harvey T, Essebier A, Vidovic D, GladwynNg I, McDonald H, Burne T, Heng J, Richards L, Gronostajski R, and Piper M (2016) Transcriptional regulation of intermediate progenitor cell generation during hippocampal development. //Development//. 143: 4620-4630; DOI: [[http://dx.doi.org/10.1242/dev.140681|10.1242/dev.140681]]
  * Patrick R, Horin C, Kobe B, Le Cao KA and Bodén M (2016) Prediction of kinase-specific phosphorylation sites through an integrative model of protein context and sequence. BBA - //Proteins and Proteomics//. 1864(11):1599-608. DOI:[[http://dx.doi.org/10.1016/j.bbapap.2016.08.001|10.1016/j.bbapap.2016.08.001]].
  * Essebier A, Vera Wolf P, Cao MD, Carroll BJ, Balasubramanian S, Bodén M (2016) Statistical enrichment of epigenetic states around triplet repeats that undergo expansions. //Frontiers in Neuroscience//. DOI: [[http://dx.doi.org/10.3389/fnins.2016.00092|10.3389/fnins.2016.00092]] (Open access)
  * Cao MD, Allison L, Dix TI and Bodén M (2016) Robust Estimation of Evolutionary Distances with Information Theory. //Molecular Biology and Evolution//. 33 (5): 1349-1357. DOI: [[http://dx.doi.org/10.1093/molbev/msw019|10.1093/molbev/msw019]]
  * Huang W, Johnston W, Bodén M and Gillam EM (2016) ReX - A suite of computational tools for the design, visualization and analysis of chimeric protein libraries. //BioTechniques//. Vol. 60, No. 2, pp. 91–94. DOI: [[http://dx.doi.org/10.2144/000114381|10.2144/000114381]].
  * Oyarzun P, Ellis JJ, Gonzalez-Galarza FF, Jones AR, Middleton D, Bodén M and Kobe B (2015) A bioinformatics tool for epitope-based vaccine design that accounts for human ethnic diversity: Application to emerging infectious diseases. //Vaccine//. 33(10):1267-73. DOI: [[http://dx.doi.org/10.1016/j.vaccine.2015.01.040| 10.1016/j.vaccine.2015.01.040]]
  * Patrick R, Le Cao KA, Kobe B and Bodén M (2015) PhosphoPICK: Modelling cellular context to map kinase-substrate phosphorylation events. //Bioinformatics//. 31(3):382-389. DOI: [[http://dx.doi.org/10.1093/bioinformatics/btu663|10.1093/bioinformatics/btu663]]
  * Cao MD, Balasubramanian S and Bodén M (2015) Sequencing technologies and tools for short tandem repeat variation detection, //Briefings in Bioinformatics//. 16 (2): 193-204. DOI:[[http://dx.doi.org/10.1093/bib/bbu001|10.1093/bib/bbu001]].
  * Rona G, Borsos M, Ellis J, Mehdi A, Christie M, Kornyei Z, Neubrandt M, Toth J, Bozoky Z, Buday L, Madarasz E, Bodén M, Kobe B and Vertessy B (2014). Dynamics of re-constitution of the human nuclear proteome after cell division is regulated by NLS-adjacent phosphorylation. //Cell Cycle//. 13(22). DOI: [[http://dx.doi.org/10.4161/15384101.2014.960740|10.4161/15384101.2014.960740]] (author eprint)
  * Mehdi AM, Patrick R, Bailey TL and Bodén M (2014) Predicting the dynamics of protein abundance, //Molecular & Cellular Proteomics//. May;13(5):1330-40. DOI:[[http://dx.doi.org/10.1074/mcp.M113.033076|10.1074/mcp.M113.033076]].
  * Cao MD, Tasker E, Willadsen K, Imelfort M, Vishwanathan S, Sureshkumar S, Balasubramanian S and Bodén M (2014) //Inferring Short Tandem Repeat Variation from Paired-End Short Reads, Nucleic Acids Research//. Feb;42(3):e16. DOI: [[http://dx.doi.org/10.1093/nar/gkt1313|10.1093/nar/gkt1313]].
  * Mazgut J, Tino P, Bodén M and Yan H (2014) //Dimensionality Reduction and Topographic Mapping of Binary Tensors, Pattern Analysis and Applications//. 17(3):497-515. DOI: [[http://dx.doi.org/10.1007/s10044-013-0317-y|10.1007/s10044-013-0317-y]].
  * Chang C-W, Counago RLM, Williams S, Bodén M and Kobe B (2013) The distribution of different classes of nuclear localization signals (NLSs) in different organisms and the utilization of the minor NLS-binding site in plant nuclear import factor importin-alpha, //Plant Signaling & Behavior//, 8(10). DOI: [[http://dx.doi.org/10.4161/psb.25976|10.4161/psb.25976]].
  * Chang C-W, Counago RLM, Williams S, Bodén M and Kobe B (2013) //Distinctive Conformation of Minor Site-Specific Nuclear Localization Signals Bound to Importin-alpha, Traffic//. Nov;14(11):1144-54. DOI: [[http://dx.doi.org/10.1111/tra.12098|10.1111/tra.12098]]
  * Oyarzun P, Ellis JJ, Bodén M and Kobe B (2013) PREDIVAC: //CD4+ T-cell epitope prediction for vaccine design that covers 95% of human HLA class II DR protein diversity, BMC Bioinformatics//, 14:52. DOI: [[http://dx.doi.org/10.1186/1471-2105-14-52|10.1186/1471-2105-14-52]]
  * Willadsen K, Cao MD, Wiles J, Balasubramanian SK and Bodén M (2013) //Repeat-encoded poly-Q tracts show statistical commonalities across species, BMC Genomics// 14:76. DOI: [[http://dx.doi.org/10.1186/1471-2164-14-76|10.1186/1471-2164-14-76]]
  * Mehdi A, Sehgal S, Kobe B, Bailey TL and Bodén M (2013) DLocalMotif: //A discriminative approach for discovering local motifs in protein sequences, Bioinformatics//, 29(1):39-46. DOI: [[http://dx.doi.org/10.1093/bioinformatics/bts654|10.1093/bioinformatics/bts654]].
  * Chang C-W, Counago RLM, Williams S, Bodén M and Kobe B (2012) //Crystal Structure of Rice Importin-alpha and Structural Basis of its Interaction with Plant-Specific Nuclear Localization Signals, The Plant Cell//, 24(12):5074-88. doi: [[http://dx.doi.org/10.1105/tpc.112.104422|10.1105/tpc.112.104422]]. (Access)
  * Patrick R, Le Cao K-A, Davis M, Kobe B and Bodén M (2012) Mapping the stabilome: a novel computational method for classifying metabolic protein stability. //BMC Systems Biol// 6(1):60. (Open access.)
  * Kobe B & Bodén M (2012) Computational modelling of linear motif-mediated protein interactions. //Curr Top Med Chem//. 12 (14), 1553-1561 [PMID: 22827524]
  * Willadsen K, Mohamad N, and Bodén M (2012) NSort/DB: An intra-nuclear compartment protein database. //Genomics, Proteomics & Bioinformatics// 10(4):226-229. DOI: [[http://dx.doi.org/10.1016/j.gpb.2012.07.001|10.1016/j.gpb.2012.07.001]]
  * Madala PK, Fairlie DP and Bodén M (2012) Matching cavities in G protein-coupled receptors to infer ligand-binding sites. //Journal of Chemical Information & Modeling//. 52(5):1401-10.
  * Bauer DC, Willadsen K, Buske FA, Le Cao K-A, Bailey TL, Dellaire G and Bodén M (2011) Sorting the nuclear proteome. //Bioinformatics//. 27(13):i7-14. (Open access.)
  * Mehdi A, Sehgal S, Kobe B, Bailey TL and Bodén M (2011) A probabilistic model of nuclear import of proteins. //Bioinformatics//. 27(9):1239-46. 
  * Marfori M, Mynott A, Ellis JJ, Mehdi AM, Saunders NF, Curmi PM, Forwood JK, Bodén M and Kobe B (2011) Molecular basis for specificity of nuclear import and prediction of nuclear localization. //Biochim Biophys Acta// 1813(9):1562-77.
  * You L, Brusic VL, Gallagher M and Bodén M (2010) Using Gaussian process with test rejection to detect T-cell epitopes in pathogen genomes. //IEEE/ACM Trans Comp Biol Bioinformatics// 7(4):741-751.
  * Mohamad N and Bodén M (2010) The proteins of intra-nuclear bodies: a data-driven analysis of sequence, interaction and expression. //BMC Systems Biology// 4:44. (Open access.)
  * Buske FA, Bodén M, Bauer DC and Bailey TL (2010) Assigning roles to DNA regulatory motifs using comparative genomics. //Bioinformatics// 26:860-6. (Open access.)
  * Bauer DC, Buske FA, Bailey TL and Bodén M (2010) Predicting SUMOylation sites in developmental transcription factors of Drosophila melanogaster. //Neurocomputing//. 73(13-15): 2300-2307.
  * Bodén M, Dellaire G, Burrage K and Bailey TL (2010) //A Bayesian network model of proteins' association with Promyelocytic leukemia (PML) nuclear bodies//. J Comp Biol 17(4): 617-30.
  * Bailey T, Bodén M, Whitington T and Machanick P (2010) The value of position-specific priors in motif discovery using MEME. //BMC Bioinformatics// 11:179. (Open access.)
  * Buske FA, Maetschke S and Bodén M (2009) It's about time: signal recognition in staged models of protein translocation. //Pattern Recognit// 42(4): 567-574.
  * Buske FA, Thier R, Gillam EM and Bodén M (2009) In silico characterisation of protein chimeras: relating sequence and function within the same fold. //Proteins// 77(1), pp. 111-120.
  * Bailey TL, Bodén M, Buske FA, Frith M, Grant CE, Clementi L, Ren J, Li WW and Noble WS (2009) MEME Suite: tools for motif discovery and searching. //Nucleic Acids Res// 37:W202-8. (Open access.)
  * Bodén M and Teasdale RD (2008) //Determining nucleolar association from sequence by leveraging protein-protein interactions//. J Comp Biol 15(3): 291-304.
  * Bodén M and Bailey TL (2008) Associating transcription factor binding site motifs with target GO terms and target genes. //Nucleic Acids Res// 36(12): 4108-4117. (Open access.)
  * Bodén M and Bodén M (2007) Evolving spelling exercises to suit individual student needs. //Appl Soft Comput// 7: 126-135.
  * Suksawatchon J, Lursinsap C and Bodén M (2007) //Computing the reversal distance between genomes in the presence of multi-gene families via binary integer programming//. J Bioinformatics Comp Biol 5(1): 117-133.
  * Hawkins J, Mahony D, Maetschke S, Wakabayashi M, Teasdale RD and Bodén M (2007) Identifying novel peroxisomal proteins. //Proteins// 69(3): 606-616.
  * Hawkins J, Davis L and Bodén M (2007) //Predicting nuclear localization//. J Proteome Res 6(4): 1402-1409.
  * Bauer, D., Bodén M., Thier, R. and Gillam, E. M. STAR: Predicting recombination sites from amino acid sequence. //BMC Bioinformatics//, 7:437, 2006. (Open access.)
  * Bodén, M. and Bailey, T. L. Identifying sequence regions undergoing conformational change via predicted continuum secondary structure. //Bioinformatics//. 22(15): 1809-1814, 2006. (Open access.)
  * Bodén, M., Yuan, Z. and Bailey, T. L. Prediction of protein continuum secondary structure with probabilistic models based on NMR solved structures. //BMC Bioinformatics//. 7:68, 2006. (Open access.)
  * Yuan, Z., Zhang, F., Davis, M. J., Bodén  M. and Teasdale, R. D. Predicting the solvent accessibility of transmembrane residues from protein sequence. //Journal of Proteome Research//. 5(5):1063-1070, 2006.
  * Hawkins, J. and Bodén  M. Detecting and sorting targeting peptides with recurrent networks and support vector machines. //Journal of Bioinformatics and Computational Biology//, 4(1), 2006.
  * Bodén, M. and Hawkins, J. Prediction of subcellular localisation using sequence-biased recurrent networks. //Bioinformatics//. 21(10):2279-2286, 2005.
  * Hawkins, J. and Bodén  M. The applicability of recurrent neural networks for biological sequence analysis. //IEEE/ACM Transactions on Computational Biology and Bioinformatics//, 2(3): 243-253, 2005.
  * Bodén, M. and Hawkins, J. Improved access to sequential motifs: A note on the architectural bias of recurrent networks. //IEEE Transactions on Neural Networks//. 16(2), 2005.
  * Bodén, M. //Generalization by symbolic abstraction in cascaded recurrent networks, Neurocomputing//, 57, pp. 87-104, 2004.
  * Bodén, M. and Blair, A. Learning the dynamics of embedded clauses, Applied Intelligence: //Special issue on natural language and machine learning//, 19(1/2), pp. 51-63, 2003.
  * Bodén, M. and Wiles, J. On learning context free and context sensitive languages, //IEEE Transactions on Neural Networks//. 13(2), pp. 491-493, 2002.
  * Bodén, M. and Wiles, J., Context-free and context-sensitive dynamics in recurrent neural networks, //Connection Science//, 12 (3/4), pp. 197-210, 2000.
  * Bodén, M. and Niklasson, L., Semantic systematicity and context in connectionist Networks, //Connection Science//, 12 (2), pp. 111-142, 2000.

Refereed conference papers (since 2000) 
====

  * Mazgut J, Tino P, Bodén M and Yan H (2010) Multilinear Decomposition and Topographic Mapping of Binary Tensors Artificial Neural Networks - ICANN 2010. K Diamantaras, W Duch, LS Iliadis (Eds.). //Lecture Notes in Computer Science//, Vol. 6352. pp. 317-326. Springer-Verlag, 2010.
  * Arieshanti I, Bodén M, Maetschke S and Buske F (2009) Detecting sequence and structure homology via an integrative kernel: A case-study in recognizing enzymes. //Proceedings of the IEEE Symposium on Computational Intelligence in Bioinformatics and Computational Biology//, pp. 46-52.
  * Bodén M (2008) Predicting nucleolar proteins using support-vector machines. //Proceedings of the Asia-Pacific Bioinformatics Conference APBC 2008//, Brazma, A., Miyano S. and Akutsu, T. (eds.) pp. 19-28, Imperial College Press.
  * Bauer D, Buske FA and Bodén M (2008) Predicting SUMOylation Sites. //Proceedings of the Third IAPR International Conference on Pattern Recognition in Bioinformatics// (PRIB 2008), Chetty M, Ngom A and Ahmad S (eds.), Springer Lecture Notes in Computer Science, vol. 5265, pp. 28-40.
  * You L, Zhang P, Bodén M and Brusic VL (2007) Understanding Prediction Systems for HLA-Binding Peptides and T-cell Epitope Identification. //Proceedings of the 2nd IAPR Workshop on Pattern Recognition in Bioinformatics//, LNBI 4774, pp. 337-348, Singapore, Springer Verlag.
  * Buske F and Bodén M (2007) Decoupling signal recognition from sequence models of protein secretion. //Proceedings of 2007 International Symposium on Computational Models for Life Sciences CMLS07//, Pham, T. and Zhou, X. (eds), pp. 147-156, American Institute of Physics.
  * Dufton L and Bodén M (2007) Reducing the number of support vectors to allay inefficiency of large-scale models in computational biology. //Proceedings of 2007 International Symposium on Computational Models for Life Sciences CMLS07//, Pham T and Zhou X (eds), pp. 340-348, American Institute of Physics.
  * Maetschke S, Gallagher M and Bodén M (2007) A Comparison of Sequence Kernels for Localization Prediction of Transmembrane Proteins. //Proceedings of the IEEE Symposium on Computational Intelligence in Bioinformatics and Computational Biology//, pp. 367-372.
  * Davis, L., Hawkins, J., Maetschke, S. and Bodén  M., Comparing SVM sequence kernels: A protein subcellular localization theme. //In Proceedings of the Workshop on Intelligent Systems for Bioinformatics//, CRPIT (vol. 73), 2006.
  * Maetschke, S., Bodén  M. and Gallagher, M., Higher order HMMs for Localization Prediction of Transmembrane Proteins. //In Proceedings of the Workshop on Intelligent Systems for Bioinformatics//, CRPIT (vol. 73), 2006.
  * Hawkins, J. and Bodén  M., Multi-stage Redundancy Reduction: Effective Utilisation of Small Protein Data Sets. //In Proceedings of the Workshop on Intelligent Systems for Bioinformatics//, CRPIT (vol. 73), 2006.
  * Bodén, M. and Hawkins, J. Evolving discriminative motifs for recognizing proteins imported to the peroxisome via the PTS2 pathway. //In Proceedings of the Congress on Evolutionary Computation//. 2006.
  * Bauer, D., Bodén  M., Thier, R. and Yuan, Z. Predicting structural disruption of proteins caused by crossover. //In Proceedings of the IEEE Symposium on Computational Intelligence in Bioinformatics and Computational Biology//, pp. 514-520. San Diego, 2005.
  * Hawkins, J. and Bodén  M. Predicting Peroxisomal proteins. //In Proceedings of the IEEE Symposium on Computational Intelligence in Bioinformatics and Computational Biology//, pp. 469-474. San Diego, 2005.
  * Suksawatchon, J., Lursinsap, C. and Bodén, M., Heuristic Algorithm for Computing Reversal Distance with Multi-Gene Families via Binary Integer Programming. //In Proceedings of the IEEE Symposium on Computational Intelligence in Bioinformatics and Computational Biology//, pp. 187-193. San Diego, 2005.
  * Wakabayashi, M., Hawkins, J., Maetschke, S. and Bodén  M., Exploiting sequence dependencies in the prediction of peroxisomal proteins. //In Intelligent Data Engineering and Automated Learning// - IDEAL 2005, pp. 454-461, 2005.
  * Bodén, M. and Hawkins, J. Detecting residues in targeting peptides. //In Proceedings of the Asia-Pacific Bioinformatics Conference//. pp. 131-140. Imperial College Press. Singapore. 2005.
  * Maetschke, S., Towsey, M. and Bodén  M. BLOMAP: An encoding of amino acids which improves signal peptide cleavage prediction. //In Proceedings of the Asia-Pacific Bioinformatics Conference//. pp. 141-150. Imperial College Press. Singapore. 2005.
  * Bodén, M. Using evolutionary noise to improve prediction of rapidly evolving targeting peptides. //In Proceedings of the 2003 Congress on Evolutionary Computation// (CEC 2003), Canberra, 2003.
  * Bodén, M. Generalization by structural properties from sparse nested symbolic data. In Proceedings of ESANN 2002 -- //Special session on Perspectives on Learning with Recurrent Networks//, pp. 377-382, 2002.
  * Bodén, M., Jacobsson, H. and Ziemke, T. (2000), Evolving context-free language predictors. In GECCO-2000: //Proceedings of the Genetic and Evolutionary Computation Conference//.

Book chapters (since 2000) 
====

  * Zaugg J, Gumulya Y, Gillam EMJ and Bodén M. (2014) Computational Tools for Directed Evolution: A Comparison of Prospective and Retrospective Strategies. Ackerley et al. (eds). //Methods Mol Biol//. 2014;1179:315-33. DOI: [[http://dx.doi.org/10.1007/978-1-4939-1053-3_21|10.1007/978-1-4939-1053-3_21]]
  * Tino P, Hammer B and Bodén M (2007) Markovian bias of neural-based architectures with feedback connections. //Perspectives of Neural-Symbolic Integration, Hitzler and Hammer (eds)//, pp. 95-133. Springer Verlag.
  * Wiles, J., Blair, A. and Bodén  M. (2001) Representation Beyond Finite States: //Alternatives to Push-Down Automata, in A Field Guide to Dynamical Recurrent Networks//, Kolen, J. F. and Kremer, S. C. (eds.), IEEE Press, pp. 129-142.
